content/build/release-notes.md

```diff
@@ -184,183 +184,185 @@ The full release note for this release is available
 
 ### New
 
-* The `buildx build` command supports new `--attest` flag, along with
+- The `buildx build` command supports new `--attest` flag, along with
   shorthands `--sbom` and `--provenance`, for adding attestations for your
-  current build. {% include github_issue.md repo="docker/buildx" number="1412" %}
-  {% include github_issue.md repo="docker/buildx" number="1475" %}
-    * `--attest type=sbom` or `--sbom=true` adds [SBOM attestations](attestations/sbom.md).
-    * `--attest type=provenance` or `--provenance=true` adds [SLSA provenance attestation](attestations/slsa-provenance.md)
-    * When creating OCI images, a minimal provenance attestation is included
-      with the image by default.
-* When building with BuildKit that supports provenance attestations Buildx will
+  current build. [docker/buildx#1412](https://github.com/docker/buildx/issues/1412)
+  [docker/buildx#1475](https://github.com/docker/buildx/issues/1475)
+  - `--attest type=sbom` or `--sbom=true` adds [SBOM attestations](attestations/sbom.md).
+  - `--attest type=provenance` or `--provenance=true` adds [SLSA provenance attestation](attestations/slsa-provenance.md)
+  - When creating OCI images, a minimal provenance attestation is included
+    with the image by default.
+- When building with BuildKit that supports provenance attestations Buildx will
   automatically share the version control information of your build context, so
   it can be shown in provenance for later debugging. Previously this only
   happened when building from a Git URL directly. To opt-out of this behavior
   you can set `BUILDX_GIT_INFO=0`. Optionally you can also automatically define
-  labels with VCS info by setting `BUILDX_GIT_LABELS=1`. {% include github_issue.md repo="docker/buildx" number="1462" %}
-  {% include github_issue.md repo="docker/buildx" number="1297" %} {% include github_issue.md repo="docker/buildx" number="1341" %}
-  {% include github_issue.md repo="docker/buildx" number="1468" %} {% include github_issue.md repo="docker/buildx" number="1477" %}
-* Named contexts with `--build-context` now support `oci-layout://` protocol
+  labels with VCS info by setting `BUILDX_GIT_LABELS=1`. [docker/buildx#1462](https://github.com/docker/buildx/issues/1462)
```

Comment:
Some formatting issues with the links: https://deploy-preview-18116--docsdocker.netlify.app/build/release-notes/#0100